src/scripts/main.js

```diff
@@ -1,7 +1,104 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+import Game from '../modules/Game.class';
 
-// Write your code here
+const game = new Game();
+
+const button = document.querySelector('.start');
+
+const gameField = document.querySelector('.game-field');
+const gameScore = document.querySelector('.game-score');
+
+const messageStart = document.querySelector('.message-start');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
```

Comment:
Ensure that all DOM elements you're trying to select do actually exist in the HTML and have the correct class names. If any of these selectors fail, the rest of the script may not work as expected.